src/logger/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+set(INCROOT ${INCROOT}/logger)
+set(SRCROOT ${SRCROOT}/logger)
+
+set(SRC
+	${SRC}
+    ${SRCROOT}/Logger.cpp
```

Comment:
You used the `tab` character for indentation in this file, but used spaces for this line.
Is this normal?

Reply:
It wasn't normal, fixed in last commit.